File: docs/node-mjs/playwright/cookie.mjs
import { chromium } from 'playwright';  // or 'firefox', 'webkit'

(async () => {
  const browser = await chromium.launch();
  const context = await browser.newContext();
  const page = await context.newPage();

  await page.goto('https://example.com');          // Navigate to website
  // Optional: perform login or other steps here

  const cookies = await context.cookies();         // Fetch all cookies  [oai_citation:0‡Playwright](https://playwright.dev/docs/api/class-browsercontext?utm_source=chatgpt.com)
  console.log('Cookies:', cookies);

  await browser.close();
})();

File: docs/node-mjs/playwright/cookie.py
from playwright.async_api import async_playwright
import json

async with async_playwright() as pw:
    browser = await pw.chromium.launch()
    context = await browser.new_context()
    page = await context.new_page()

    await page.goto('https://example.com')
    cookies = await context.cookies()  # Fetch cookies  [oai_citation:3‡stably.ai](https://www.stably.ai/blog/mastering-cookie-management-in-playwright?utm_source=chatgpt.com)
    print("Cookies:", cookies)

    # Save cookies to file
    with open('cookies.json', 'w') as f:
        json.dump(cookies, f)

    # To reuse in a fresh context:
    new_context = await browser.new_context()
    with open('cookies.json', 'r') as f:
        saved = json.load(f)
    await new_context.add_cookies(saved)  # Reuse ()

    new_page = await new_context.new_page()
    await new_page.goto('https://example.com')
    # You're now using the saved session
    await browser.close()
